ui.apps/src/main/content/jcr_root/apps/acs-commons/config.author/org.apache.sling.jcr.repoinit.RepositoryInitializer-varworkflow-acls.config

```diff
@@ -0,0 +1,13 @@
+scripts=[
+# these users and ACLs are only necessary on author
+create service user acs-commons-workflow-remover-service with path system/acs-commons
+set principal ACL for acs-commons-workflow-remover-service
+    allow jcr:read, rep:write on /var/workflow/instances
+end
+
+create service user acs-commons-workflowpackagemanager-service with path system/acs-commons
+set principal ACL for acs-commons-workflowpackagemanager-service
+    allow jcr:read on /var/workflow/packages
+end
+"
+]
```

Comment:
```suggestion
scripts=["
# these users and ACLs are only necessary on author
create service user acs-commons-workflow-remover-service with path system/acs-commons
set principal ACL for acs-commons-workflow-remover-service
    allow jcr:read, rep:write on /var/workflow/instances
end

create service user acs-commons-workflowpackagemanager-service with path system/acs-commons
set principal ACL for acs-commons-workflowpackagemanager-service
    allow jcr:read on /var/workflow/packages
end
"
]
```

Reply:
Missing opening quote

Reply:
Fixed in https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/2523/commits/71adee822a8aec9ff5163d8e50c3f0a2865ad9a7.